quandl/model/database.py

```diff
@@ -58,12 +59,10 @@ def bulk_download_to_file(self, file_or_folder_path, **options):
 
     def _bulk_download_path(self):
         url = self.default_path() + '/data'
-        url = Util.constructed_path(url, {'id': self.database_code})
+        url = Util.constructed_path(url, {'id': self.code})
         return url
 
     def datasets(self, **options):
-        params = {'database_code': self.database_code, 'query': '', 'page': 1}
+        params = {'database_code': self.code, 'query': '', 'page': 1}
         options = Util.merge_options('params', params, **options)
-        return Dataset.all(**options)
-
-from .dataset import Dataset
+        return quandl.model.dataset.Dataset.all(**options)
```

Comment:
Since this is fixed let's remove the flake8 exclusion I put in to avoid this warning when you rebase off of master.